locales/de.json

```diff
@@ -20,10 +20,10 @@
 	"driver-select-vehicle-type.motorbike": "🏍 Roller / Motorrad",
 	"driver-select-vehicle-type.car": "🚗 Auto",
 	"driver-select-vehicle-type.error_only_known_type": "Unbekannte Auswahl, bitte wähle Auto oder Roller / Motorrad aus.",
-	"driver-explain-checkins.text": "Fast fertig, noch ein paar Dinge zur Beachtung. Telegramm hat keine Echtzeit-GPS-Standortunterstützung. Also jedes Mal, wenn du deinen Standort ändert, checke ein. Oder checke jede Stunde ein, wenn du bereit bist neue Aufträge zu akzeptieren. Das wird uns helfen, Passagiere im Umkreis zu finden. Tippe \"Weiter\".",
+	"driver-explain-checkins.text": "Fast fertig, bevor es losgeht gibt es noch ein paar Dinge zu beachten. Telegramm hat keine Echtzeit-GPS-Standortunterstützung. Also jedes Mal, wenn du deinen Standort änderst, checke ein. Oder checke jede Stunde ein, wenn du bereit bist neue Aufträge zu akzeptieren. Das wird uns helfen, Passagiere im Umkreis zu finden. Tippe \"Weiter\".",
 	"driver-explain-checkins.next": "Weiter",
 	"driver-request-location.provide_location": "Bitte teile deinen Standort, damit Passagiere im Umkreis dich finden können.",
-	"driver-explain-whats-next.text": "Alles Klar! Du wirst über Passagiere und Aufträge im Umreis benachrichtigt. Halte diese App offen und checke jede Stunde ein, und wenn du deinen Standort änderst. Tippe \"Weiter\".",
+	"driver-explain-whats-next.text": "Alles Klar! Du wirst über Passagiere und Aufträge im Umreis benachrichtigt. Lass LibreTaxi geöffnet und checke jede Stunde ein, und wenn du deinen Standort änderst. Tippe \"Weiter\".",
 	"driver-explain-whats-next.next": "Weiter",
 	"driver-index.checkin": "📍 Check-In",
 	"driver-index.mute": "🔕 Stumm",
```

Comment:
line 36: change "👌 Danke für das Einchecken!" to "👌 Danke fürs Einchecken!"
change "Bitte überprüfe den obigen Standort, ist er nicht korrekt, checke erneut ein und tippe auf Büroklammer-Standort (anstatt eine Taste zu  drücken)." to  "Bitte überprüfe den obigen Standort - ist er nicht korrekt, checke erneut ein und tippe auf Büroklammer-Standort (anstatt eine Taste zu drücken)."